packages/editor-ui/src/api/settings.ts

```diff
@@ -10,3 +11,11 @@ export async function submitPersonalizationSurvey(context: IRestApiContext, para
 	await makeRestApiRequest(context, 'POST', '/user-survey', params as unknown as IDataObject);
 }
 
+export async function getPromptsData(instanceId: string): Promise<IN8nPrompt> {
+	return await get(TEMPLATES_BASE_URL, '/prompts', {}, {'n8n-instance-id': instanceId});
```

Comment:
If we have static object, it will throw an error on the POST request since the DB is not pulled with n8n-instance-id's. For testing purpose, you can use from test1 - test20 as an instance id. Even numbers (test2, test4 ...) response with showValueSurvey: true.